db/migrate/20170427130832_rename_section_edition_section_id_to_section_uuid.rb

```diff
@@ -0,0 +1,13 @@
+class RenameSectionEditionSectionIdToSectionUuid < Mongoid::Migration
+  def self.up
+    SectionEdition.all.each do |section_edition|
+      section_edition.rename(:section_id, :section_uuid)
+    end
+  end
+
+  def self.down
+    SectionEdition.all.each do |section_edition|
+      section_edition.rename(:section_uuid, :section_id)
+    end
+  end
+end
```

Comment:
Minor: Are you confident that this will be quick enough running against production data? Did you consider using a bulk update command like the one in [RenameManualRecordEditionsRemovedDocumentIdsToRemovedSectionIds](https://github.com/alphagov/manuals-publisher/blob/master/db/migrate/20170329103706_rename_manual_record_editions_removed_document_ids_to_removed_section_ids.rb)?